sharepoint/sharepoint-ps/sharepoint-online/Get-SPORestrictedSiteCreation.md

```diff
@@ -0,0 +1,77 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-sporestrictedsitecreation
+applicable: SharePoint Online
+title: Get-SPORestrictedSiteCreation
+schema: 2.0.0
+author: vgaddam-pm
+ms.author: vgaddam
+ms.reviewer:
+---
+
+# Get-SPORestrictedSiteCreation
+
+## SYNOPSIS
+
+This cmdlet allows SharePoint administrators to check the current configuration of the restricted site creation feature.
+
+## SYNTAX
+
+```powershell
+Get-SPORestrictedSiteCreation [-SiteType <RestrictedSiteCreationSiteType>]
+```
+
+## DESCRIPTION
+
+This cmdlet obtains the current configuration information for the restricted site creation feature, including whether it is enabled, the current mode, and the current policies.
+
+> [!Important]
+>You must use version 16.0.25513 (published November 2024) or later of the [SharePoint Online Management Shell](https://www.microsoft.com/en-us/download/details.aspx?id=35588&msockid=3bc54cadb1bd648039325ff4b0b5653a) for these commands to function properly. Earlier versions do not have the current list of site types and will not operate correctly.
```

Comment:
>16.0.25513 [](http://example.com/codeflow?start=22&length=10)

also as mentioned in the other PR: we don't yet know if this will be the actual November build.